tests/validation/test_highlights.py

```diff
@@ -0,0 +1,366 @@
+                                                                 'hostName': 'string'},
+                                       'certificateConfig': {'certificates': []}},
+                                 headers={'Accept': 'application/json'})
+        pytest.DURATION = 5
+        pytest.JSON = {
+            "nonExistingObjects": [],
+            "relationships": [],
+            "result": [
+                {
+                    "events": [],
+                    "key": {
+                        "adapterKind": "HighlightsMP",
+                        "identifiers": [],
+                        "name": "fist",
+                        "objectKind": "Growing Object"
+                    },
+                    "metrics": [
+                        {
+                            "key": "stable number",
+                            "numberValue": 42.0,
+                            "timestamp": 1666796497855
+                        }
+                    ],
+                    "properties": [
+                        {
+                            "key": "stable_property",
+                            "stringValue": "I am stable",
+                            "timestamp": 1666813121048
+                        }
+                    ]
+                }
+            ]
+        }
+
+    def test_no_highlight(self):
+        response = TestObject()
+        response.is_success = True
+        response.text = json.dumps(pytest.JSON)
+        normal_collection = LongCollectionBundle(5, 25)
+        # Add ten identical collections
+        for collection in range(10):
+            collection_bundle = CollectionBundle(pytest.REQUEST, copy.deepcopy(response), pytest.DURATION,
+                                                 pytest.CONTAINER_STATISTICS)
+            collection_bundle.collection_number = collection
+            normal_collection.add(collection_bundle)
+
+        highlight = normal_collection.validate()
+        assert len(highlight.messages) == 0
+
+    def test_highlight_for_growing_unique_objects(self):
+        response = TestObject()
+        response.is_success = True
+        _json = copy.deepcopy(pytest.JSON)
+        growing_unique_objects_overtime = LongCollectionBundle(5, 25)
+        for collection in range(10):
+            response.text = json.dumps(_json)
+            collection_bundle = CollectionBundle(pytest.REQUEST, copy.deepcopy(response), pytest.DURATION,
+                                                 pytest.CONTAINER_STATISTICS)
+            collection_bundle.collection_number = collection
+            growing_unique_objects_overtime.add(collection_bundle)
+            _json["result"][0]["key"]["name"] = f"{collection}"
+
+        highlight = highlight_object_growth(growing_unique_objects_overtime.long_collection_statistics)
+        assert (ResultLevel.WARNING,
+                "Object of type HighlightsMP::Growing Object displayed growth of 9.65%") in highlight.messages
```

Comment:
A few things here that apply to all the messages:
* "Object of type" should be "Objects of type", since we are talking about all objects of the given type, not a singular one
* Should add single quotes around types, keys, etc
* I think we talked about adding a time period to the unit, so it's clear what the growth rate is measuring.
```
"Objects of type 'HighlightsMP::Growing Object' had a growth rate of 9.65% per ???"
```